packages/clay-management-toolbar/src/ClayManagementToolbar.soy

```diff
@@ -553,6 +569,72 @@
 	</nav>
 {/template}
 
+/**
+ * This renders the search main input.
+ */
+{template .searchInput}
+	{@param spritemap: string}
+	{@param? _handleCloseMobileSearchClick: any}
+	{@param? _handleSearchSearchClick: any}
+	{@param? contentRenderer: string}
+	{@param? disabled: bool}
+	{@param? onlySearch: bool}
+	{@param? searchInputName: string}
+	{@param? searchValue: string}
+	{@param? showAdvancedSearch: bool}
+
+	<div class="input-group">
+		<div class="input-group-item">
+			{let $inputAttributes kind="attributes"}
+				aria-label="Search"
+				{if $disabled}
+					disabled
+				{/if}
+
+				class="form-control input-group-inset input-group-inset-after"
+				name="{$searchInputName}"
+				placeholder="Search for..."
+				ref="search"
+				type="text"
+				value="{$searchValue}"
+			{/let}
+
+			<input {$inputAttributes} />
+
+			<span class="input-group-inset-item input-group-inset-item-after">
+				{if not $onlySearch}
+					{call ClayButton.render}
+						{param elementClasses: 'd-none' /}
+						{param events: ['click': $_handleCloseMobileSearchClick] /}
+						{param icon: 'times' /}
+						{param spritemap: $spritemap /}
+						{param style: 'unstyled' /}
+					{/call}
+				{/if}
+
+				{call ClayButton.render}
+					{param disabled: $disabled /}
+					{param events: ['click': $_handleSearchSearchClick] /}
+					{param icon: 'search' /}
+					{param ref: 'searchButton' /}
+					{param spritemap: $spritemap /}
+					{param style: 'unstyled' /}
+					{param type: 'submit' /}
+				{/call}
+
+				{if $showAdvancedSearch}
+					{call ClayDropdown.render}
+						{param contentRenderer: $contentRenderer /}
+						{param icon: 'angle-down' /}
+						{param items: [] /}
```

Comment:
Hey @carloslancha, is this expected?

Reply:
Yep, since I don't want items to be an optional parameter we need to do this.